Dockerfile.netkan

```diff
@@ -5,6 +5,6 @@ RUN useradd -ms /bin/bash netkan
 USER netkan
 WORKDIR /home/netkan
 ADD netkan.exe .
-ENTRYPOINT /usr/bin/mono netkan.exe --queues $QUEUES \
+ENTRYPOINT /usr/bin/mono netkan.exe --game $GAME --queues $QUEUES \
```

Comment:
Could NetKAN when operating in 'Queue' mode, look for an attribute of 'game'? We could add this to the scheduler/webhooks in advance of this PR.

Reply:
That's not out of the question, but we'll still need the command line parameter so we can inflate netkans outside the infra.